library/src/main/java/com/digi/xbee/api/AbstractXBeeDevice.java

```diff
@@ -3280,6 +3290,62 @@ private boolean enterATCommandMode() throws InvalidOperatingModeException, Timeo
 		return false;
 	}
 	
+	/**
+	 * Exits AT command mode. The XBee device has to be in command mode.
+	 * 
+	 * @throws InterfaceNotOpenException if this device connection is not open.
+	 */
+	private void exitATCommandMode() {
+		if (!connectionInterface.isOpen())
+			throw new InterfaceNotOpenException();
+		
+		try {
+			// Send the exit ('ATCN\r') command.
+			connectionInterface.writeData(COMMAND_MODE_EXIT.getBytes());
+			// Wait the guard time.
+			Thread.sleep(DEFAULT_GUARD_TIME);
+		} catch (IOException e) {
+			logger.error(e.getMessage(), e);
+		} catch (InterruptedException e) {
+			logger.error(e.getMessage(), e);
+		}
```

Comment:
NP: these exceptions can be compacted in a multi-catch block to not repeat code (the same below).